rct229/data/ashrae_90_1_table_G3_6.json

```diff
@@ -1,134 +1,134 @@
 {
     "building_exteriors":[
         {
-            "building_exterior_type": "Uncovered parking lots and drives",
+            "exterior_lighting_area": "Uncovered parking lots and drives",
```

Comment:
For naming *this* property in the json file, we should follow the table in the standard as closely as possible (as apposed to our schema). The table title is "Performance Rating Method Lighting Power Densities for Building Exteriors", so I think your original `building_exterior_type` was a closer fit.

Reply:
Okay, I'll revert the JSON.